test/RawDatasetDatablock.js

```diff
@@ -164,37 +164,69 @@ describe("RawDatasetDatablock: Test Datablocks and their relation to raw Dataset
         res.body.datablocks[0].should.have
           .property("dataFileList")
           .and.be.instanceof(Array)
-          .and.to.have.length(
-            TestData.OrigDataBlockCorrect.dataFileList.length,
-          );
+          .and.to.have.length(TestData.DataBlockCorrect.dataFileList.length);
       });
   });
 
-  it("Should fetch some filenames from the new dataset", async () => {
-    var fields = {
-      datasetId: datasetPid,
-      filenameExp: "B410",
-    };
-    var limits = {
-      skip: 0,
-      limit: 20,
-    };
+  // NOTE: Currently fullquery on dataset does not allow/support searching for fields on a related datablock.
```

Comment:
I would remove the comment

Reply:
Yes we can do the cleanup later but I wanted to have it here just to know why this test is commented.

Reply:
If the comment is meaningful for the refactoring, let's keep it and we will remove it when we are ready to release

Reply:
yes, sorry I meant to remove the whole commented block

Reply:
No problem!!! Good point, let's remove it

Reply:
Done

Reply:
thanks!